R/db-interface.R

```diff
@@ -42,11 +42,11 @@
 copy_dm_to <- function(dest, dm, ...,
                        types = NULL, overwrite = NULL,
                        indexes = NULL, unique_indexes = NULL,
-                       set_key_constraints = TRUE, unique_table_names = FALSE,
-                       table_names = NULL,
+                       set_key_constraints = TRUE,
+                       table_names = repair_table_names_for_db(src_tbls(dm), schema, temporary),
```

Comment:
Defaulting to an unexported function looks odd. Can we keep the `NULL` default and use `repair_table_names_for_db()` internally?